src/client/components/vision/camera/view.tsx

```diff
@@ -0,0 +1,61 @@
+import { autorun } from 'mobx'
+import { action } from 'mobx'
+import { observer } from 'mobx-react'
+import * as React from 'react'
+import { Component } from 'react'
+
+import { CameraViewModel } from './view_model'
+
+@observer
+export class CameraView extends Component<{ viewModel: CameraViewModel }> {
+  private destroy: () => void = () => {}
+
+  componentDidMount() {
+    this.destroy = autorun(() => this.renderScene(), {
+      scheduler: requestAnimationFrame,
+    })
+  }
+
+  componentWillUnmount() {
+    this.destroy()
+  }
+
+  render() {
+
+    const { width, height } = this.props.viewModel
+
+    const aspectRatio = width / height
+    const percentage = 60
+
+    return (
+      <canvas style={{
+        display: 'inline',
```

Comment:
Move non-computed styling to CSS classes, but I'm not sure what you are looking to achieve with this style anyway?

I suspect its for keeping camera views next to each other, in which case the parent should be responsible for that (e.g. with `display: flex`).

Reply:
👍